apps/server/src/modules/tldraw/entities/tldraw-drawing.entity.spec.ts

```diff
@@ -12,16 +12,24 @@ describe('tldraw entity', () => {
 				const tldraw = new TldrawDrawing({
 					docName: 'test',
 					version: 'v1_tst',
-					value: 'bindatamock',
+					value: Buffer.from('bindatamock'),
 					_id: 'test-id',
 					clock: 0,
 					action: 'update',
+					part: 1,
 				});
 				expect(tldraw).toBeInstanceOf(TldrawDrawing);
 			});
 
 			it('should throw with empty docName', () => {
-				const call = () => new TldrawDrawing({ docName: '', version: 'v1_tst', value: 'bindatamock', _id: 'test-id' });
+				const call = () =>
+					new TldrawDrawing({
```

Comment:
Can you please move the new TldrawDrawing behind a test factory with default values than is placed in modules/tldraw/testing/* . Can you also work with bson package new ObjectId().toHexString() for each id if it is a MongoDB objectId instead of generic string.


This factory can use existing production factories if it exists but pass default testing values to it.

Reply:
done